portfolio/src/main/webapp/hiking.html

```diff
@@ -39,7 +39,7 @@
     const PREFIX = "hiking";
 
     allTileBoxs(PREFIX, TILES);
```

Comment:
please go through this pull request and fix typos like this.  Boxs => Boxes, there are quite a few.  May seem nitpicky but this kind of easily-fixable tech debt accumulates and can make debugging/searching code later on very frustrating.